cmd/node/node.go

```diff
@@ -600,10 +600,16 @@ func (app *App) initServices(ctx context.Context, poetClients []activation.PoetP
 	if trtlCfg.BadBeaconVoteDelayLayers == 0 {
 		trtlCfg.BadBeaconVoteDelayLayers = app.Config.LayersPerEpoch
 	}
-	trtl, err := tortoise.Recover(app.cachedDB, beaconProtocol,
-		tortoise.WithContext(ctx),
```

Comment:
I think this functional option might have been accidentally dropped.

Reply:
not accidental, i need to remove WithContext completely, it was used long time ago